.github/workflows/tests.yml

```diff
@@ -32,15 +32,16 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Set up Python
-      uses: actions/setup-python@v2
+      uses: actions/setup-python@v2.1.4
```

Comment:
What is the reason to pin this version strictly? I guess updates on the v2 branch should not break the builds.

Reply:
They where breaking the builds before in a version newer then the specified one. I was not yet able to check if v2 is enough, but at least it seems to work for now.

Reply:
👍🏾

Reply:
Let's try v2 here.